Summary: Whole-Genome Sequencing Accurately Identifies Resistance to Extended-Spectrum β-Lactams for Major Gram-Negative Bacterial Pathogens

Authors:

**Ayesha Noor ( @AyeshaNoor), Cateline Atieno Ouma (@Cateline), Mahesh Rani Kamilus (), Nada Esmael Sleim (@Eddy27)**

# Video Presentation:

https://www.linkedin.com/posts/cateline-ouma-6470aa1b2\_as-part-of-the-hackbio-cancer-internship-activity-7238264755675664384-LY1Y/?utm\_source=share&utm\_medium=member\_android

**Introduction**

In modern microbiology, the ability to swiftly and accurately identify antimicrobial resistance (AMR) is paramount, particularly in the face of rising infections caused by resistant pathogens. Whole-genome sequencing (WGS) integration emerges as a transformative approach in the fight against AMR. This literature review is on the application of WGS to predict AMR in gram- negative bacteria isolated from cancer patients with neutropenic fever—a group particularly vulnerable to severe infections. Through a comprehensive analysis of  a study done by Sheburne et al., (2019), we aim to highlight how the power of genomic technology can reshape our understanding and management of antimicrobial resistance, especially in clinical settings.

# Objective and Methodology

The main objective of the study was to determine the accuracy of using whole-genome sequencing (WGS) to identify resistance to extended-spectrum β-lactams in major gram-negative bacterial pathogens.

Methodology:

1.      Specimen Selection- The study analyzed 90 bloodstream isolates from the four most common gram-negative bacteria responsible for bloodstream infections in neutropenic patients: *Escherichia coli, Klebsiella pneumoniae, Pseudomonas aeruginosa,* and *Enterobacter cloacae.* Bacterial strains were isolated from high-risk cancer patients who were neutropenic (neutrophil count <500/µL).

2.      Antimicrobial Susceptibility Testing- Broth microdilution (BMD) assays were performed to determine the minimum inhibitory concentrations (MIC) for four β-lactams commonly used in empirical treatment: ceftazidime, cefepime, piperacillin-tazobactam (P/T), and meropenem.

3.      Whole-Genome Sequencing (WGS)- Illumina MiSeq sequencing technology was employed to generate WGS data from the isolates.

A custom database of AMR protein sequences was constructed by merging data from the Antibiotic Resistance Database (ARDB) and Comprehensive Antibiotic Resistance Database (CARD), focusing on β-lactamase genes and their mutations.

4.      Data Analysis-The WGS data were used to identify gene presence/absence and specific mutations associated with AMR mechanisms. Predictions of resistance to β-lactams were generated based on the identified resistance genes and mutations, and the predictions were compared against the phenotypic resistance profiles obtained from BMD.

5.      Statistical Assessment- Sensitivity, specificity, and predictive values of the WGS method were calculated by comparing genotypic resistance predictions to the phenotypic outcomes established by traditional BMD testing and commercial methods.

Statistical tests, including the McNemar test for agreement and Cohen’s kappa for interrater agreement, were conducted to assess the performance of the WGS approach.

**Results**

***Mechanisms of Resistance***

Resistance mechanisms were primarily due to:

![https://lh7-rt.googleusercontent.com/slidesz/AGV_vUehSOAF2RVQ8h2-6PG4zgfJvEFcB9qIKgy0aDpX60VKOjaNwQqh8lCOM2GZ_HaQNoGjZ6DU9kXGvZOjdMLKn81BXGGrebierE_tRvCz-H208zU8I6ezh6tsAMDvpWrk9CrD4ThfO8CKe4wjwbbJdg81Z-azDSg=s2048?key=H2PvYYm4i1Jcw2hyCMJgrA](file:///C:/Users/ADMINI~1/AppData/Local/Temp/msohtmlclip1/01/clip_image002.gif)

·         Exogenous β-lactamase acquisition in E. coli and K. pneumoniae (98% of predicted instances).

·         Chromosomal mutations in P. aeruginosa and E. cloacae, with significant contributions from mutations in regulatory genes and porin function.

Among the 360 combinations of isolates and β-lactams tested, WGS identified a total of 133 predicted instances of antimicrobial resistance (AMR) to the four β-lactams of interest.

Only 87 out of 133 (approximately 65%) of these predicted resistance events would have been detected using a typical PCR-based approach.

***Diagnostic Performance***
![image](https://github.com/user-attachments/assets/f9e79e25-243d-4472-aa4a-19858927be19)


The sensitivity of the WGS method for predicting AMR was 0.87 (87%).

The specificity was exceptionally high at 0.98 (98%).

The positive predictive value was 0.97 (97%), while the negative predictive value was 0.91 (91%).

Comparatively, the WGS method demonstrated a significantly higher positive predictive value than commercial methods (0.97 vs 0.92; P = .025).

**Agreement with Reference Method:**

There was 93% agreement (336 instances) between the WGS predictions and the phenotypic susceptibility results obtained via (BMD).

Most disagreements (13 instances) arose from an inability to detect a genotypic mechanism for phenotypic resistance to P/T, while no disagreements were observed for meropenem.

**Conclusion**

The WGS approach performed comparably to reference methods and showed superior predictive capacity in classifying AMR compared to results from commercial susceptibility testing methods. As WGS technology advances, it has the potential to become a standard tool in clinical microbiology

**Reference**
